lapis2-docs/src/content/docs/tutorials/maintainer-tutorials/start-lapis-and-silo.mdx

```diff
@@ -0,0 +1,219 @@
+We recommend using those Docker images, so in this tutorial, we explain how to use them.
+
+### Prerequisites
+
+-   You have [Docker](https://www.docker.com/) installed.
+-   Some knowledge on how to use Docker and Docker Compose.
+-   Make sure you have the latest Docker images:
+
+```shell
+docker pull ghcr.io/genspectrum/lapis-v2
+docker pull ghcr.io/genspectrum/lapis-silo
+```
+
+-   Create a directory for the example:
+
+```shell
+mkdir ~/lapisExample
+```
+
+### Writing Configuration
+
+Both LAPIS and SILO need to know which metadata columns are available in the dataset.
+Furthermore, you need to define which column acts as primary key
+and which column should be used to generate partitions in SILO.
+Also, LAPIS is configured to be an open instance, meaning that the underlying data requires no visibility restrictions.
+
+##### `~/lapisExample/config/database_config.yaml`
+
+```yaml
+schema:
+    instanceName: testInstance
+    metadata:
+        - name: primaryKey
+          type: string
+        - name: date
+          type: date
+        - name: region
+          type: string
+          generateIndex: true
+        - name: country
+          type: string
+          generateIndex: true
+        - name: division
+          type: string
+          generateIndex: true
+        - name: pangoLineage
+          type: pango_lineage
+        - name: age
+          type: int
+        - name: qc_value
+          type: float
+    opennessLevel: OPEN
+    primaryKey: primaryKey
+    dateToSortBy: date
+    partitionBy: pangoLineage
+```
+
+:::tip
+See the [config reference](../../references/configuration) for a full specification.
+:::
+
+### Starting SILO Preprocessing
+
+Download the example dataset from the [end-to-end tests](https://github.com/GenSpectrum/LAPIS/tree/main/siloLapisTests/testData):
+
+-   pangolineage_alias.json
+-   reference_genomes.json
+-   small_metadata_set.tsv
+-   all fasta files for the sequences
+
+SILO expects fasta files (possibly compressed via zstandard or xz)
+in the same directory with naming scheme `nuc_<sequence_name>.fasta` for nucleotide sequences
+or `gene_<sequence_name>.fasta` for amino acid sequences.
+The `sequence_names`s have to match the names defined in the `reference_genomes.json`.
+
+Put those files into the folder `~/lapisExample/data/`.
+
+Now SILO needs to know where it can find those files.
+You have to provide a "preprocessing config" for that.
+Note that you need to provide the paths where the files will be stored in the Docker container.
+Filenames are relative to the input directory.
+Since you don't provide the input directory explicitly, SILO will fall back to the default `/data`.
+
+##### `~/lapisExample/config/preprocessing_config.yaml`
+
+```yaml
+metadataFilename: 'small_metadata_set.tsv'
+pangoLineageDefinitionFilename: 'pangolineage_alias.json'
+referenceGenomeFilename: 'reference_genomes.json'
+```
+
+Start the SILO Docker container with the options:
```

Comment:
This sentence somehow does not match the bullet points below. Could you clarify this?

Reply:
I adapted it.